etc/config/c++.amazon.properties

```diff
@@ -840,6 +840,18 @@ compiler.arm1021.exe=/opt/compiler-explorer/arm/gcc-arm-none-eabi-10-2020-q4-maj
 compiler.arm1021.name=ARM gcc 10.2.1 (none)
 compiler.arm1021.semver=10.2.1
 compiler.arm1021.supportsBinary=false
+compiler.arm1031_07.exe=/opt/compiler-explorer/arm/gcc-arm-none-eabi-10.3-2021.07/bin/arm-none-eabi-gcc
+compiler.arm1031_07.name=ARM gcc 10.3.1 (2021.07 none)
```

Comment:
I like that there is the ARM version (2021.07 none) here. Maybe we should apply the same for the other ones. What do you think @partouf ?

Reply:
only that specific version of that compiler for Arm has that. It was released due to a security workaround